docs/Terminology.rst

```diff
@@ -0,0 +1,25 @@
+.. _Observation_Terminology:
+
+***************
+Terminology
+***************
+
+Scheduling
+==================
+Determined during the python in astronomy conference:
+
+* **observing block** (OB): the minimum schedulable block
```

Comment:
There's [a case to be made](https://github.com/astropy/astroplan/pull/151#issuecomment-224250861) that the observing block is distinct from a minimum schedulable block. I think OB needs a more generic definition, like maybe "a single or group of visits to one or multiple targets, with associated instrument configurations and observational constraints."